lib/views/post/widgets/vote_button.dart

```diff
@@ -0,0 +1,170 @@
+import 'package:flutter/material.dart';
+import 'package:pr0gramm/constants/vote_constants.dart';
+import 'package:pr0gramm/entities/enums/vote.dart';
+import 'package:pr0gramm/entities/enums/vote_button_type.dart';
+import 'package:pr0gramm/services/vote_animation_service.dart';
+
+class VoteButton extends StatefulWidget {
+  final VoteAnimationService animationService;
+  final VoteButtonType type;
+  final bool disabled;
+
+  VoteButton({
+    Key key,
+    @required this.type,
+    this.disabled = false,
+    this.animationService,
```

Comment:
Isn't this required too?

Reply:
will be obsolete when splitting into multiple Widgets